lib/modules/platform/github/index.ts

```diff
@@ -116,14 +116,15 @@ export async function detectGhe(token: string): Promise<void> {
 
 export async function initPlatform({
   endpoint,
-  token,
+  token: token_,
```

Comment:
```suggestion
  token: originalToken,
```

don't use leading underscrore